First commits (#2)
cisagov/li-pca-app: CONTRIBUTING.md

If you want to report a bug or request a new feature, the most direct
method is to [create an

> issue](https://github.com/cisagov/skeleton-generic/issues) in this

Reviewer: Not sure why the repo name was changed back to `skeleton-generic` here, but it should reference this repo:
```suggestion
issue](https://github.com/cisagov/li-pca-app/issues) in this
```

repository.  We recommend that you first search through existing
issues (both open and closed) to check if your particular issue has
already been reported.  If it has then you might want to add a comment
to the existing issue.  If it hasn't then feel free to create a new
one.

## Branches ##

If you are a developer assigned to the PCA team, when creating a branch,
reference the corresponding ticket in JIRA as a prefix to the branch name.

For example: PCADEV-123_some_feature_branch

When pushing from your local repo to a remote, push first to your
remote branch and then create a pull request to merge back into develop.

> Pushes directly to develop is not allowed.

Reviewer: Correct grammar:
```suggestion
Pushes directly to develop are not allowed.
```

## Pull requests ##

If you choose to [submit a pull
request](https://github.com/cisagov/skeleton-generic/pulls), you will
notice that our continuous integration (CI) system runs a fairly
extensive set of linters and syntax checkers.  Your pull request may
fail these checks, and that's OK.  If you want you can stop there and
wait for us to make the necessary corrections to ensure your code

> passes the CI checks.

Reviewer: Again, you managed to put the repo name back to the skeleton, so that should be corrected.  Also, I'm guessing that your editor is reformatting these lines- in general, you want to minimize changes to skeleton text because it will make life simpler when future changes come in from upstream.  So I recommend putting this back the way that it was:
```suggestion
request](https://github.com/cisagov/li-pca-app/pulls),
you will notice that our continuous integration (CI) system runs a
fairly extensive set of linters, syntax checkers, system, and unit tests.
Your pull request may fail these checks, and that's OK.  If you want
you can stop there and wait for us to make the necessary corrections
to ensure your code passes the CI checks.
```

commands:

```console
cd skeleton-generic
pyenv virtualenv <python_version_to_use> skeleton-generic

> pyenv local skeleton-generic

Reviewer: Another one:
```suggestion
cd li-pca-app
pyenv virtualenv <python_version_to_use> li-pca-app
pyenv local li-pca-app
```

you attempt to commit.  If you want to run the checks against the
entire repo, just execute `pre-commit run --all-files`.

### Running unit and system tests ###

In addition to the pre-commit checks the CI system will run the suite
of unit and system tests that are included with this project.  To run
these tests locally execute `pytest` from the root of the project.

We encourage any updates to these tests to improve the overall code
coverage.  If your pull request adds new functionality we would
appreciate it if you extend existing test cases, or add new ones to
exercise the newly added code.

> 

Reviewer: Why did you delete this block of text?

Author: This must have gotten removed unintentionally. I'll get it added back.
